packages/reports/addon/components/common-actions/schedule.ts

```diff
@@ -134,88 +144,102 @@ export default class ScheduleActionComponent extends Component<Args> {
       return true;
     }
 
-    return !(this.localDeliveryRule?.hasDirtyAttributes && this.isRuleValid);
+    return !(this.currentDisplayedRule?.hasDirtyAttributes && this.isRuleValid);
   }
 
   /**
-   * @property {Boolean} attemptedSave
+   * @action addNewRule
    */
-  @tracked attemptedSave = false;
+  @action
+  addNewRule() {
+    this.currentDisplayedRule = this._createNewDeliveryRule();
+    if (this.currentDisplayedRule.name === '') {
+      this.currentDisplayedRule.name = this.defaultName;
```

Comment:
Instead of saving the default rule, can we just display the `defaultName` in the case the model name is blank/empty?